app/src/main/java/com/concordium/wallet/ui/MainActivity.kt

```diff
@@ -61,7 +61,11 @@ class MainActivity : BaseActivity(), IdentityStatusDelegate by IdentityStatusDel
 
         initializeViews()
 
-        intent?.data?.let { if (it.toString().startsWith("wc")) wcUri = it.toString() }
+        intent?.data?.let {
+            if (it.toString().startsWith("wc") || it.toString()
```

Comment:
These lines are duplicated twice below, so create a helper function to perform the check instead.

I don't really understand why it needs to set `wcUri` in both places in the intents, but I guess it's easier to just keep that as is.